infra/gravity/ops_center.go

```diff
@@ -0,0 +1,159 @@
+package gravity
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	errClusterNotFound = errors.New("cluster not found")
```

Comment:
please use explicit trace.NotFoundError instead of custom variable here and in similar cases